mutiny-wasm/src/models.rs

```diff
@@ -338,3 +342,90 @@ impl AuthProfile {
         JsValue::from_serde(&serde_json::to_value(&self.used_services).unwrap()).unwrap()
     }
 }
+
+#[derive(Debug, Serialize, Deserialize, Clone, Copy, Eq, PartialEq, Hash)]
+#[wasm_bindgen]
+pub enum TagKing {
```

Comment:
TagKind? or the king of the tags

Reply:
I also misspell kind as king, I have no idea why.